lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -35,6 +35,11 @@ const UIStrings = {
    * */
   crossoriginWarning: 'A preconnect <link> was found for "{securityOrigin}" but was not used ' +
     'by the browser. Check that you are using the `crossorigin` attribute properly.',
+  /**
+   * @description A warning message that is shown when found more than 8 preconnected links
+   * */
+  tooManyPreconnectLinksWarning: 'More than 8 preconnect links were found. ' +
```

Comment:
8 -> 3 here and in the description

Reply:
warning at 3 or 4? Please note the sentence structure here - i.e `_more_ than 3`.

From your comment https://github.com/GoogleChrome/lighthouse/pull/9903#issuecomment-564796338, I think you meant to suggest doing `More than 2` (which means we warn at 3).